docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Security considerations
+parent: Operating System
+nav_order: 330
+---
+## Security considerations
+
+As a networked device, reachable from any network, the attack surface of the BitBox Base needs to be minimized as much as possible. This is why we use our open-source [BitBox App](https://github.com/digitalbitbox/bitbox-wallet-app/) as the dedicated user interface for node management. By default, only one port is exposed on the Base to communicate over an end-to-end encrypted channel with the App, providing all functionality.
+
+### Pairing Base with BitBox App
+The first step is to safely pair the BitBox Base and App. To achieve this, the Base announces itself on the local network using mDNS and the Middlware exposes provides an API endpoint to initiate a secure connection. The Base is automatically detected by the App and announced within the user interface. 
+
+At first, only a single public API endpoint for pairing is available that allows the two components to establish an encrypted communication channel. To rule out a man-in-the-middle attack, a confirmation code is shown both within the App and on the Base and needs to be confirmed manually. With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the full API functionality is available to the App.
```

Comment:
> I added some examples.

(I'm creating a new comment thread here since I can't find a way to reply on the [previous one](https://github.com/digitalbitbox/bitbox-base/pull/50#discussion_r292364187).)

Which commit changed the wording to add examples here? I couldn't find any reference to such changes in b4da43738f851dab12275bde35e5093d32da8f4c..

Reply:
pushed now.